<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1050</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1050</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>cipher and decipher</string>
  </property>
  <property name="windowIcon">
   <iconset resource="sources.qrc">
    <normaloff>:/Images/window icon.png</normaloff>:/Images/window icon.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>40</width>
    <height>120</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1050</width>
      <height>700</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="sources.qrc">:/Images/mainwindow.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textInput">
    <property name="geometry">
     <rect>
      <x>62</x>
      <y>245</y>
      <width>350</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPlainTextEdit{
color:white;
background:transparent;border:none;
	font: 700 12pt &quot;Segoe UI&quot;;
}</string>
    </property>
    <property name="plainText">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Enter text here</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textOutput">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>245</y>
      <width>350</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPlainTextEdit{
color:white;
background:transparent;border:none;
	font: 700 12pt &quot;Segoe UI&quot;;
}</string>
    </property>
    <property name="plainText">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Output will appear here</string>
    </property>
   </widget>
   <widget class="QLabel" name="cipherName">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>260</y>
      <width>124</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:white;
	font: 700 10pt &quot;Segoe UI&quot;;
}</string>
    </property>
    <property name="text">
     <string>affine</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="cipherBtn">
    <property name="geometry">
     <rect>
      <x>466</x>
      <y>317</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:transparent;border:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="decipherBtn">
    <property name="geometry">
     <rect>
      <x>466</x>
      <y>360</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:transparent;border:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="affine">
    <property name="geometry">
     <rect>
      <x>57</x>
      <y>25</y>
      <width>138</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:transparent;border:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="atbash">
    <property name="geometry">
     <rect>
      <x>217</x>
      <y>25</y>
      <width>138</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:transparent;border:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="baconian">
    <property name="geometry">
     <rect>
      <x>376</x>
      <y>25</y>
      <width>138</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:transparent;border:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="morceCode">
    <property name="geometry">
     <rect>
      <x>536</x>
      <y>25</y>
      <width>138</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:transparent;border:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="polybiusSquare">
    <property name="geometry">
     <rect>
      <x>695</x>
      <y>25</y>
      <width>138</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:transparent;border:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="RailFence">
    <property name="geometry">
     <rect>
      <x>856</x>
      <y>25</y>
      <width>138</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:transparent;border:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="simpleSubstitution">
    <property name="geometry">
     <rect>
      <x>376</x>
      <y>106</y>
      <width>138</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:transparent;border:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="vignere">
    <property name="geometry">
     <rect>
      <x>535</x>
      <y>106</y>
      <width>138</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:transparent;border:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="xorCipher">
    <property name="geometry">
     <rect>
      <x>695</x>
      <y>106</y>
      <width>138</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:transparent;border:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="caesar">
    <property name="geometry">
     <rect>
      <x>217</x>
      <y>106</y>
      <width>138</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:transparent;border:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="sources.qrc"/>
 </resources>
 <connections/>
</ui>
